package.nls.fr.json

```diff
@@ -0,0 +1,25 @@
+{
+  "command.regenerateIcons.title": "Appliquer Icônes Personnalisation",
+  "command.ngPreset.title": "Basculer Angular Préréglé (Niveau d'espace de travail)",
+  "command.jsPreset.title": "Basculer JS Officiel Préréglé (Niveau de l'utilisateur)",
+  "command.tsPreset.title": "Basculer TS Officiel Préréglé (Niveau de l'utilisateur)",
+  "command.jsonPreset.title": "Basculer JSON Officiel Préréglé (Niveau de l'utilisateur)",
+  "command.hideFoldersPreset.title": "Basculer la visibilité des icônes des dossiers (Niveau de l'utilisateur)",
```

Comment:
I think the whole translation should be reviewed by a French native before being published. 

After consulting with my Belgian colleague he said to me that some parts of the texts are not real French (as I suspected). Of course any French man would understand it but that doesn't mean it's properly structured. In my opinion, we should not publish any translation until it's properly validated by a native.